src/main/java/seedu/module/logic/commands/RecurCommand.java

```diff
@@ -0,0 +1,115 @@
+package seedu.module.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.module.model.Model.PREDICATE_SHOW_ALL_TASKS;
+
+import java.util.List;
+
+import seedu.module.commons.core.Messages;
+import seedu.module.commons.core.index.Index;
+import seedu.module.logic.commands.exceptions.CommandException;
+import seedu.module.model.Model;
+import seedu.module.model.task.Deadline;
+import seedu.module.model.task.DoneStatus;
+import seedu.module.model.task.Recurrence;
+import seedu.module.model.task.Task;
+
+
+/**
+ * Makes a task repeat at a given interval of
+ */
+public class RecurCommand extends Command {
```

Comment:
I have a few questions:
1) Wouldn't the edit command suffice for changing the recurrence of a task?
2) If we decide to keep the recur command, is there a way to remove recurrence without using edit command? I think users may be confused as to why the way to remove recurrence is not obvious.

Reply:
Actually, for 1, scratch the question. We also have a tag command and edit command that can handle tag changes.

Reply:
Yes. I think there should be a notRecur command to remove the recurrence of the Task. I think i will work on that Monday after clearing other assignments haha